### ISSUE TITLE
Cannot automate any alloy metals using blends via refined storage

### ISSUE THREAD

**Issue by jeviuss** [scalda/Foolcraft_3#116]
## Modpack Information
- [x] I am running the latest version of FoolCraft
- [x] I am running FoolCraft via Curse/Twitch Launcher
- [x] I can reproduce this issue consistently
     - [ ] In single player
     - [x] In multiplayer
- [x] I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I'm having the same problem.
- [ ] I am crashing and can provide my crash report(s)
- [x] I have not altered the modpack [if you have, note the removed mods/added mods/changes in Additional Information]

## Issue Description
Refined storage creates blends (such as Lumium) and sends it into a crafter pointing into an alloy smelter and/or induction smelter along with sand. Using the crafting pattern will create the blend but not put it into either alloy machine to create the actual ingot.

## Reproduce Steps
Basically exactly what I said above

## Additional Information
I've tried switching all the options around such as locking flux slots, input/output, removing the slag from the results in the crafting pattern for the induction furnace, replacing machines and crafters, and resetting the entire refined storage system. I think I may be doing something wrong but I can't tell what.

**Comment by Mrbysco** [comment]
Is your alloy machine set to alloy mode or furnace mode. Also I've been able to use the induction smelter

**Comment by jeviuss** [comment]
I tried it with it set to both, as well as the induction smelter with and without the flux slot locked.

**Comment by scopiplays** [comment]
try disabling the oredict for the crafting pattern.  I have had problems automating alloys and unchecking that box has solved the problem for me.
